volatility3/framework/plugins/windows/netscan.py

```diff
@@ -13,7 +13,7 @@
 from volatility3.framework.symbols.windows import versions
 from volatility3.framework.symbols.windows.extensions import network
 from volatility3.plugins import timeliner
-from volatility3.plugins.windows import info, poolscanner
+from volatility3.plugins.windows import info, poolscanner, verinfo
```

Comment:
This needs a corresponding `VersionRequirement` in the plugin.

Reply:
Sure, for which version? As far as I can see verinfo does not define a version tuple yet. (https://github.com/volatilityfoundation/volatility3/blob/4e50402f1123519f71d7d8878c9344341d7557b6/volatility3/framework/plugins/windows/verinfo.py)

Reply:
Ah, good spot, I bumped the wrong value, so you helped me fix an error too!  5:D  Should be resolved in the branch now...  5:)

Reply:
Yep, so (1, 0, 0) would be the version to depend on please.  5:)

Reply:
done!